rayon-core/src/join/test.rs

```diff
@@ -44,7 +43,7 @@ fn sort_in_pool() {
     let mut rng = XorShiftRng::from_seed([0, 1, 2, 3]);
     let mut data: Vec<_> = (0..12 * 1024).map(|_| rng.next_u32()).collect();
 
-    let pool = ThreadPool::new(Configuration::new()).unwrap();
+    let pool = Configuration::new().init().unwrap();
```

Comment:
(bike-shedding) It doesn't seem great that this call says nothing about being a thread pool -- though this was true of `::build()` too.  Maybe it's OK for folks who would call it like `rayon::Configuration::new().init()` so the context isn't entirely lost.